demux/demux_cue.c

```diff
@@ -44,18 +44,14 @@
 
 #define OPT_BASE_STRUCT struct demux_cue_opts
 struct demux_cue_opts {
```

Comment:
empty struct? Probably can remove it now.

Reply:
Indeed. You only need it in `options.c` for the redirect to work. Removed.